sdcm/stress_thread.py

```diff
@@ -106,6 +106,10 @@ def create_stress_cmd(self, node, keyspace_idx):
             self.profile = cs_profile
             self.keyspace_name = keyspace_name
 
+        if 'duration' in stress_cmd:
+            duration = self.loader_set.params.get("test_duration") - 20
```

Comment:
I guess we always need the stress command smaller in some degree from then the test duration.
for example to calculate start the prepare stage, but it can't be a fixed number, and I'm not sure we can easily calculate it.

the only thing I can think of, is to lookup the history of the prepare, for that job, and reduce that.

Reply:
problem is, i think 20 minutes would not be enough for most cases...